sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Plugin.java

```diff
@@ -22,27 +22,18 @@
 import org.apache.commons.lang.StringUtils;
 
 import java.util.Arrays;
-import java.util.List;
-
-public class Plugin extends Artifact {
-
-  private String name;
-  private String description;
-  private String homepageUrl;
-  private String license;
-  private String organization;
-  private String organizationUrl;
-  private String termsConditionsUrl;
-  private String category;
-  private String issueTrackerUrl;
-  private String sourcesUrl;
+
+public class Plugin extends Component {
```

Comment:
This is a breaking change (return type of inherited methods changes). As we discussed, probably no big deal for us. But this implies a new major release as well. Double-check with a SQ backend.